zeppelin-interpreter/src/main/java/com/teragrep/zep_01/conf/ZeppelinConfiguration.java

```diff
@@ -1025,7 +1025,8 @@ public enum ConfVars {
     ZEPPELIN_SESSION_CHECK_INTERVAL("zeppelin.session.check_interval", 60 * 10 * 1000),
     ZEPPELIN_NOTE_FILE_EXCLUDE_FIELDS("zeppelin.note.file.exclude.fields", ""),
     ZEPPELIN_RESOURCEPOOL_MAX_OBJECT_SIZE("zeppelin.resourcepool.max.object.size", 10 * 1024),
-    ZEPPELIN_RESOURCEPOOL_MAX_OBJECT_COUNT("zeppelin.resourcepool.max.object.count", 1000);
+    ZEPPELIN_RESOURCEPOOL_MAX_OBJECT_COUNT("zeppelin.resourcepool.max.object.count", 1000),
+    ZEPPELIN_ANNOUNCEMENT("zeppelin.announcement","");
```

Comment:
A space is missing after the comma
```suggestion
    ZEPPELIN_ANNOUNCEMENT("zeppelin.announcement", "");
```

Reply:
Fixed in commit [f05402f](https://github.com/teragrep/zep_01/pull/272/commits/f05402fc361a930b7075b29265fe5dd345678ca9)